cmd/create.go

```diff
@@ -83,31 +84,31 @@ func runCreateCmd(cmd *cobra.Command, args []string) {
 	deplDir := filepath.Join(outputDir, deplName)
 	checkErr(modulewriter.WriteDeployment(dc, deplDir, overwriteDeployment))
 
-	fmt.Println("To deploy your infrastructure please run:")
-	fmt.Println()
-	fmt.Printf(boldGreen("%s deploy %s\n"), execPath(), deplDir)
-	fmt.Println()
+	logging.Info("To deploy your infrastructure please run:")
+	logging.Info("")
+	logging.Info(boldGreen("%s deploy %s\n"), execPath(), deplDir)
```

Comment:
Either remove `\n` from format string  (preferable) or `logging.Info("")`. Otherwise you are doing one more newline than original code.

Reply:
Removed `\n`.